### PR TITLE
fix(questionnaire): Fixes error when trying to view a questionnaire modified by a deleted admin

### LINKED ISSUE #217: Questionnaires inaccessible when admin deleted
Questionnaires and check-in pages for an attendee within Dashboard are inaccessible if an admin who modified a model being rendered in the history card is deleted.

Cause: It appears this is a result of AuditHelper using find() to search for a user with a supplied ID that doesn't exist.

### PR DESCRIPTION
**Pull request by cbaudouinjr**
resolves #217

### CODE REVIEW

File: test/models/questionnaire_test.rb [hunk 1]
```diff
@@ -402,6 +409,13 @@ class QuestionnaireTest < ActiveSupport::TestCase
       assert_nil questionnaire.checked_in_by_id
     end
 
+    should "return no one if user who checked-in questionnaire is deleted" do
```
Comment (peterkos) [inline test/models/questionnaire_test.rb:412]: nit: Maybe should be `return nil if user...` to be consistent with previous test wording

Comment (cbaudouinjr) [reply test/models/questionnaire_test.rb:412]: Which previous test? I was going initially put nil but the previous test is
`should "return no one if not checked in" do
      questionnaire = create(:questionnaire)
      assert_nil questionnaire.checked_in_by
      assert_nil questionnaire.checked_in_by_id
    end`

Comment (peterkos) [reply test/models/questionnaire_test.rb:412]: In reference to the other test you added in this commit, sorry! not the previous in the file.
```return nil if author deleted```
`

Comment (cbaudouinjr) [reply test/models/questionnaire_test.rb]: Ah gotcha, good grab! Patched

File: test/models/questionnaire_test.rb [hunk 2]
```diff
@@ -204,6 +204,13 @@ class QuestionnaireTest < ActiveSupport::TestCase
       assert_nil questionnaire.acc_status_author
     end
 
+    should "return nil if author deleted" do
+      user = create(:user, email: "admin@example.com")
```
Comment (peterkos) [inline test/models/questionnaire_test.rb:208]: Get ready to refactor this after #241 :p
Or crap. That's gonna be my problem now...

**Pull request merged**